### PR TITLE
Add HideSearch

### PR DESCRIPTION
**Pull request by ketal178**
# Add HideSearch

## 更改内容 / Types of Changes
- 新功能 New Feature
- UI界面 User Interface

## 由这个PR解决或关闭的Issues / Issues Fixed or Closed by This PR

-  

## 检查列表 / Checklist

- [x] 我的代码符合本仓库的代码规范 My code follows the code style of this project
- [x] 我已经测试了这些更改，它们可以正常工作，既不会破坏原有任何功能(或我可以解决)，也不会影响对旧版本的支持 I have tested the changes and verified that they work and don't break anything(as well as I can manage) or drop the support for previous versions.
- [x] 我已合并了对后续工作无意义的commit并确认不会对后续维护造成困扰 I have merged commits that are meaningless for follow-up work and confirmed that they will not cause damage to follow-up maintenance

### CODE REVIEW

File: app/src/main/java/me/ketal/hook/HideTab.kt [hunk 1]
```diff
@@ -64,36 +61,31 @@ object HideTab : CommonDelayableHook("ketal_HideTab") {
...
                     tab.currentTab = if (size == 3) 2 else 3
                 }
             }
         }
     }
 
-    private fun addSettingItem(linearLayout: LinearLayout, drawable: Drawable?, label: String, clickListener: View.OnClickListener) {
+    private fun addSettingItem(linearLayout: LinearLayout, resName: String, label: String, clickListener: View.OnClickListener) {
         val ctx = linearLayout.context
         val view = View.inflate(ctx, ctx.hostLayout("b2g")!!, null) as LinearLayout
```
Comment (NextAlone) [inline app/src/main/java/me/ketal/hook/HideTab.kt:86]:
```suggestion
         val view = View.inflate(ctx, ctx.hostLayout("b2g")?, null) as LinearLayout
```

File: app/src/main/java/me/ketal/hook/HideTab.kt [hunk 2]
```diff
@@ -64,36 +61,31 @@ object HideTab : CommonDelayableHook("ketal_HideTab") {
...
         val imgView = view[0] as ImageView
         val textView = view[1] as TextView
-        imgView.setImageDrawable(drawable)
+        imgView.setImageResource(ctx.hostDrawable(resName)!!)
```
Comment (NextAlone) [inline app/src/main/java/me/ketal/hook/HideTab.kt:85]:
```suggestion
        imgView.setImageResource(ctx.hostDrawable(resName)?)
```

File: app/src/main/java/me/ketal/hook/HideTab.kt (hunk 1 above)
Comment (NextAlone) [reply app/src/main/java/me/ketal/hook/HideTab.kt:86]: 尽量不要!!

File: app/src/main/java/me/ketal/hook/HideTab.kt (hunk 2 above)
Comment (ketal178) [reply app/src/main/java/me/ketal/hook/HideTab.kt:85]: 后面在改吧，限定了qq860，不会出事的

Comment (NextAlone) [reply app/src/main/java/me/ketal/hook/HideTab.kt:85]: 直接再commit一个就行

**Review by NextAlone — APPROVED** [review]

**Review by cinit — APPROVED** [review]

**Pull request merged**